### PR TITLE
fix issue (#359)

### PR DESCRIPTION
**Pull request by Sun-Woo-Kim**
fix issue (#359)

### CODE REVIEW

File: ios/RNAdMobManager/RNAdmobNativeAdsManager.m [hunk 1]
```diff
@@ -67,12 +67,12 @@ @implementation RNAdmobNativeAdsManager
...
     if ([[config allKeys] containsObject:@"tagForChildDirectedTreatment"]) {
         NSNumber *tag = [config valueForKey:@"tagForChildDirectedTreatment"];
-        [[[GADMobileAds sharedInstance] requestConfiguration] tagForChildDirectedTreatment:tag.boolValue];
+        [[[GADMobileAds sharedInstance] requestConfiguration] setTagForChildDirectedTreatment:[NSNumber numberWithBool:tag.boolValue]];
     };
     
     if ([[config allKeys] containsObject:@"tagForUnderAgeConsent"]) {
         NSNumber *tagC = [config valueForKey:@"tagForUnderAgeConsent"];
-        [[[GADMobileAds sharedInstance] requestConfiguration] tagForUnderAgeOfConsent:tagC.boolValue];
+        [[[GADMobileAds sharedInstance] requestConfiguration] setTagForChildDirectedTreatment:[NSNumber numberWithBool:tagC.boolValue]];
```
Comment (tcaish) [inline ios/RNAdMobManager/RNAdmobNativeAdsManager.m:75]: Is there not a tagForUnderAgeConsent alternative, as well? Or does both have to be setTagForChildDirectedTreatment?

Comment (tcaish) [reply ios/RNAdMobManager/RNAdmobNativeAdsManager.m:75]: This did fix it though after running a build for iOS using Expo EAS.

**Comment by Sun-Woo-Kim** [comment]
Hi, @ammarahm-ed Please review this. Thank you!

**Review by digibuilders — APPROVED** [review]
Fixed Expo Build Error, react-native-admob-native-ads 0.6.6. PLEASE MERGE

**Pull request merged**